docs/sql-ref-syntax-ddl-drop-table.md

```diff
@@ -26,6 +26,8 @@ if the table is not `EXTERNAL` table. If the table is not present it throws an e
 
 In case of an external table, only the associated metadata information is removed from the metastore database.
 
+If the table is cached, the command uncaches the table and all its dependants such as views. Thereby the dependants should be cached again explicitly.
```

Comment:
nit: dependants -> dependents

Reply:
Thanks. I ran `grep` over all sources codes, and had found one more place:
https://github.com/apache/spark/blob/0986f16c8d2f0d648d5239958dd4bccac43890c2/sql/core/src/main/scala/org/apache/spark/sql/internal/CatalogImpl.scala#L555
I will fix it in this PR as well.